src/elements/call-out/src/index.tsx

```diff
@@ -43,44 +41,12 @@ const CallOut: React.FC<Props> = ({ title, text, icon }) => {
   return (
     <Palette
       as="div"
-      sx={{
-        borderRadius: 4,
-        paddingX: 'sm',
-        paddingY: 'sm',
-        marginTop: 'xs',
-        marginBottom: 'md',
-        variant: 'callOut.main'
-      }}
+      sx={{ variant: 'callOut.main' }}
       px={{
         backgroundColor: 'accentColor'
       }}
     >
-      {icon ? (
-        <div sx={{ display: 'flex' }}>
-          <div
-            sx={{
-              flexGrow: 0,
-              flexShrink: 0,
-              flexBasis: ({ space }: any) => space.sm,
-              marginRight: 'xs'
-            }}
-          >
-            <div
-              sx={{
-                borderRadius: '50%',
-                padding: '20%',
-                variant: 'callOut.icon'
-              }}
-            >
-              <Icon color="white" glyph={icon} />
-            </div>
-          </div>
-
-          <div>{mainText}</div>
-        </div>
```

Comment:
Is there a reason we've removed the icon? I imagine that this will be a requirement on other brands or use cases?

Reply:
I didnt saw it in the design. Should i just keep it as it is (optional) ? @domtronn ?

Reply:
I think so - It'll be part of another brands designs and we shouldn't be removing stories as these describe the functional requirements of the component, even if they're not used by all designs yet

Reply:
Reverted @domtronn